cronjobs/cnn_classify_data.py

```diff
@@ -87,6 +100,7 @@
         obsdates_tilenumbers[:,0]=obsdates
         obsdates_tilenumbers[:,1]=tilenums
```

Comment:
I think it is in this else statement that we may want to make sure that things work also with just one obsdate and tilenum (and in the for loop below)